src/platforms/php/common/integrations.mdx

```diff
@@ -8,49 +8,42 @@ redirect_from:
 
 ## Default Integrations
 
-Default integrations are integrations enabled by default that integrate into the
-standard library or the interpreter itself. They are documented so you can see
-what they do and that they can be disabled if they cause issues. To disable
-system integrations set `default_integrations => false` when calling `init()`.
+These integrations are enabled by default and integrate into the
+standard library or the interpreter itself. They are documented so you can be both aware of what they do and disable them if they cause issues.
+
+To disable system integrations, set `default_integrations => false` when calling `init()`.
 
 ### ExceptionListenerIntegration
 
 This integration catches all global uncaught exceptions and emits events when an error occurs.
 
-To do that, it ensures that Sentry's `ErrorHandler` is registered, and adds a callback to it
-as an exception listener.
+To do so, it ensures that Sentry's `ErrorHandler` is registered, and adds a callback to it as an exception listener.
 
 ### ErrorListenerIntegration
 
 This integration hooks into the global PHP `error_handler` and emits events when an error occurs.
 
-To do that, it ensures that Sentry's `ErrorHandler` is registered, and adds a callback to it
-as an error listener. By default, the `ErrorHandler` reserves 10 kilobytes of memory to handle
-fatal errors.
+To do so, it ensures that Sentry's `ErrorHandler` is registered, and adds a callback to it as an error listener. By default, the `ErrorHandler` reserves 10 kilobytes of memory to handle fatal errors.
 
-For some frameworks or projects, there are specific integrations provided both
-officially and by third-party users that automatically register the error
-handlers. In that case, please refer to their documentation.
+For some frameworks or projects, specific integrations are provided both officially and by third-party users that automatically register the error handlers. Please refer to their documentation.
 
-Also, by default `E_ALL` will be handled, you can change it by setting `error_types`
-a different constant in `init()`.
+Also, by default `E_ALL` will be handled. You can change it by setting `error_types` to a different constant in `init()`.
```

Comment:
This does not hold true anymore. We changed a while back the default of this option so that whatever is the current PHP reporting level we use it. You can still explicitly set the option to a fixed value of course

Reply:
updated to reflect that `E_ALL` defaults to the current PHP reporting level.

Reply:
I'm sorry, I probably expressed myself badly: what I meant was that while earlier the `error_types()` option defaulted to that constant, since a while it defaults to the value returned by the `error_reporting()` function which can change at runtime. So, if the user adjust the level of errors to report during the application execution by default now Sentry will react accordingly. If the user sets the option to a fixed value instead, Sentry will always log the errors of that specific type regardless of the current error reporting level

Reply:
See https://github.com/getsentry/sentry-docs/pull/3470#issuecomment-830426263 for the continuation to the discussion